test/v1/otherReliefs/retrieve/RetrieveOtherReliefsValidatorFactorySpec.scala

```diff
@@ -14,12 +14,13 @@
  * limitations under the License.
  */
 
-package v1.controllers.validators
+package v1.otherReliefs.retrieve
 
 import api.models.domain.{Nino, TaxYear}
 import api.models.errors._
 import support.UnitSpec
-import v1.models.request.retrieveOtherReliefs.RetrieveOtherReliefsRequestData
+import v1.otherReliefs.retrieve.def1.model.request.Def1_RetrieveOtherReliefsRequestData
+import v1.otherReliefs.retrieve.model.request.RetrieveOtherReliefsRequestData
 
 class RetrieveOtherReliefsValidatorFactorySpec extends UnitSpec {
```

Comment:
Again this should be `Def1_RetrieveOtherReliefsValidatorSpec` and in the def1 package. See similar comment for amend.